registry_module_integration_test.go

```diff
@@ -1323,6 +1324,60 @@ func TestRegistryModulesRead(t *testing.T) {
 	})
 }
 
+func TestRegistryModulesReadRegistry(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+	r := require.New(t)
+
+	// create an org that will be deleted later. the wskp will live here
```

Comment:
seems there is a typo in the comment. Is this comment needed? `orgTestCleanup` explains the cleanup

Reply:
Not really, I'll remove this.